test/setup-test-framework.js

```diff
@@ -0,0 +1,21 @@
+require( 'core-js/modules/es7.object.values' );
```

Comment:
This is tricky part. I had to add it explicitly here. `babel-jest` should do it instead, but apparently there is something happening that breaks it.